src/diff.ts

```diff
@@ -104,6 +104,7 @@ async function getMostRecentBranchBuild(
  * the default branch in git, and pick the most topologically recent.
  */
 async function getBaseBuildForDefaultBranch(info: BuildkiteEnvironment): Promise<BuildkiteBuild> {
+  log(`Getting base base for default branch`);
```

Comment:
```suggestion
  log(`Getting base build for default branch`);
```

Or is `base base` intentional? It makes sense to me in context but will probably be confusing otherwise